tail/tail_test.go

```diff
@@ -115,43 +121,21 @@ func TestGetEntries(t *testing.T) {
 
 	jsonLines := make([][]string, 3)
 	filenameRoot := ts.tmpdir + "/json.log"
+
+	linesChans := make(chan chan string)
+	errChan := make(chan error)
+	fileChan := make(chan string)
+	go GetEntries(ts.ctx, conf, fileChan, linesChans, errChan)
 	for i := 0; i < 3; i++ {
 		jsonLines[i] = make([]string, 3)
 		for j := 0; j < 3; j++ {
 			jsonLines[i][j] = fmt.Sprintf("{\"a\":%d}", i)
 		}
 
 		filename := filenameRoot + fmt.Sprint(i)
-		conf.Paths[i] = filename
+		fileChan <- filename
 		ts.writeFile(t, filename, strings.Join(jsonLines[i], "\n"))
-	}
```

Comment:
Not crazy about taking the below chunk of the test out, but it was definitely oriented towards the old model of returning a list of `chan string` (in this case, an empty one) instead of taking in a `chan chan string` that gets written to. Any ideas or do you think it's OK to just remove?